staging/src/k8s.io/kubectl/pkg/util/interrupt/interrupt.go

```diff
@@ -102,3 +113,11 @@ func (h *Handler) Run(fn func() error) error {
 	defer h.Close()
 	return fn()
 }
+
+// checkToAddNewLine checks if we defined the addNewLineAfterTermination with true value
+// adds a new line
+func (h *Handler) checkToAddNewLine() {
+	if h.addNewLineAfterTermination {
+		fmt.Print("\n")
```

Comment:
Although managing this via new `addNewLineAfterTermination` field makes sense and is a good idea, this `fmt.Print("\n")` creates problem. Because there are various consumers to this package and they inject their own notify mechanisms. I think, statically `fmt.Print("\n")` causes troubles.

Reply:
Thanks for taking a look at this, i'm not sure what else should i do maybe there is some other characters instead of `\n` that will do the job or not. Propose if you ok with it i ask about others opinion on regular meeting if it's not acceptable we can close this and issue or maybe find another solution for it.

Reply:
> Propose if you ok with it i ask about others opinion on regular meeting

Of course, feel free to ask.